src/screens/login/components/LoginForm.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+
+import {
+  Text,
+  View,
+  TextInput,
+  TouchableOpacity,
+  ActivityIndicator,
+} from 'react-native';
+
+import {styles} from './styles';
+import {validateLoginData} from '../validation';
+
+interface LoginFormsProps {
+  email: string;
+  setEmail: (email: string) => void;
+  password: string;
+  setPassword: (email: string) => void;
+  loading: boolean;
+  handleLoginUser: () => void;
```

Comment:
```suggestion
  onLoginUser: () => void;
```

A ideia é usar o `on` no nome da prop, e o handle no nome da função que a implementa.